SmartPlaces.Facilities/lib/OntologyMapper.Mapped/test/OntologyMapper.Mapped.Test.csproj

```diff
@@ -28,7 +28,7 @@
 		<PackageReference Include="Microsoft.Azure.DigitalTwins.Parser" Version="6.4.0-preview" />
 		<PackageReference Include="Microsoft.Extensions.Configuration" Version="6.0.1" />
 		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2" />
-		<PackageReference Include="Microsoft.SmartPlaces.Facilities.OntologyMapper" Version="0.6.4-preview" />
+		<PackageReference Include="Microsoft.SmartPlaces.Facilities.OntologyMapper" Version="0.6.5-preview" />
```

Comment:
Since you have the project reference which references this nuget, you should be able to not explicitly include this reference. Doesn't harm anything, but could save you a step.